packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -0,0 +1,120 @@
+import { app, BrowserWindow } from 'electron';
+import { autoUpdater, UpdateInfo } from 'electron-updater';
+import log from 'electron-log';
+import path = require('path');
+import { setInterval } from 'timers';
+
+
+export enum AutoUpdaterEvents {
+	CheckingForUpdate = 'checking-for-update',
+	UpdateAvailable = 'update-available',
+	UpdateNotAvailable = 'update-not-available',
+	Error = 'error',
+	DownloadProgress = 'download-progress',
+	UpdateDownloaded = 'update-downloaded',
+}
+
+const DEFAULT_UPDATE_INTERVAL = 12 * 60 * 60 * 1000;
+const INITIAL_UPDATE_STARTUP = 5 * 1000;
```

Comment:
We use camelCase for constants: https://joplinapp.org/help/dev/coding_style#use-camelcase-for-constants-in-new-code